modules/groovy-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/execution/RecipeExecutorAdaptor.groovy

```diff
@@ -44,4 +45,9 @@ class RecipeExecutorAdaptor implements RecipeExecutor {
     void removeRecipeFilter(RecipeFilter recipeFilter) {
 
     }
+
+    @Override
+    ExecutionMode getExecutionMode() {
+        return null
```

Comment:
Should we use Optional<ExecutionMode>? Or do we maybe need to be 1.7 compatible?